projects/fsastorefrontlib/src/occ/adapters/quote/occ-quote.adapter.ts

```diff
@@ -49,7 +49,7 @@ export class OccQuoteAdapter implements QuoteAdapter {
     userId: string,
     cartId: string,
     quoteAction: string,
-    body: any
+    quoteBody: any
```

Comment:
Can we try to define a type instead of using **any**?

Reply:
In both quote adapter and connector, the body or quoteContetnt types are all set to be "any" so I will leave it like this. If we chose to do refactoring then all should be changed.